src/reanimated2/hook/Hooks.ts

```diff
@@ -13,4 +14,9 @@ export function useWorkletCallback<A extends unknown[], R>(
   return useCallback(fun, deps ?? []);
 }
 
+/**
+ * `useLayoutAnimation` just returns the memoized function, so it's just a typed helper with useMemo.
+ */
+export const useLayoutAnimation = useMemo
```

Comment:
Shouldn't this be `useCallback`, since it's receiving an argument?

Reply:
Right, thanks!